docs/images.md

```diff
@@ -37,23 +39,23 @@ This worked perfectly for access to the contents of the published support docume
 
-For implementation reasons, the problems presented by students copying images into their own documents were mitigated somewhat by the caching model provided by the `ImageMap`. The situation in which this arose most acutely was in a class where the teacher was using published supports essentially as curriculum. Students would copy images from these supports into their own documents and then later they would open up their documents and the initial request for the image data would fail -- it wasn't coming from the context of a multi-class support and so the cross-class access request would fail. But then later if that same student opened up the support document that represented their curriculum, the image access would succeed. At that point, the `ImageMap` cached the association between the image id/url and its image data so any subsequent request for that image, e.g. to redraw the document in which the image access initially failed, would successfully retrieve the image data. As such, these initial failures would appear to magically fix themselves, possibly even before the user noticed a problem beyond an initial delay. (_We_ noticed, however, as each such initial failed access resulted in a Rollbar error.)
+For implementation reasons, the problems presented by students copying images into their own documents were mitigated somewhat by the caching model provided by the `ImageMap`. The situation in which this arose most acutely was in a class where the teacher was using published supports essentially as curriculum. Students would copy images from these supports into their own documents and then later they would open up their documents and the initial request for the image data would fail -- it wasn't coming from the context of a multi-class support and so the cross-class access request would fail. But then later if that same student opened up the support document that represented their curriculum, the image access would succeed. At that point, the `ImageMap` cached the association between the image ID/URL and its image data so any subsequent request for that image, e.g. to redraw the document in which the image access initially failed, would successfully retrieve the image data. As such, these initial failures would appear to magically fix themselves, possibly even before the user noticed a problem beyond an initial delay. (_We_ noticed, however, as each such initial failed access resulted in a Rollbar error.)
```

Comment:
Another block where the only real changes are pedantic (`id` to `ID` and `url` to `URL`). Sorry. I didn't realize those changes wouldn't be highlighted in some cases.